scripts/Player.gd

```diff
@@ -23,13 +23,17 @@ func _physics_process(delta):
 		direction.y += 1
 	if Input.is_action_pressed("move_down"):
 		direction.y -= 1
```

Comment:
What if this became:
```suggestion
		direction = Vector3.DOWN
```

and similarly for the others?

Reply:
The addition/subtraction is actually pretty important here because it allows the player to strafe (ex. go forward and to the right) and prevents the player from moving if opposite directions are pressed (if player presses both forward and backwards, there will be 1 added z but also 1 subtracted from z, resulting in a net zero as expected).

Reply:
You could still increment by these built-in variables..?